challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { | animal | animal.capitalize }
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.find_all { |animal| animal.downcase == "dog" }
```

Comment:
`.find_all` works but `.select` is what we're looking for here.

Reply:
They are aliases, and most folks prefer `select`.

Here's a style guide many rubyists follow: https://github.com/bbatsov/ruby-style-guide#map-find-select-reduce-size